ui/css/treeherder-navbar.css

```diff
@@ -366,9 +366,6 @@ fieldset[disabled] .btn-view-nav-closed.active {
 }
 
 @media (min-width: 960px) {
-  .btn-toggle-excluded::after {
-    content: " Excluded Jobs";
-  }
   .btn-excluded::after {
```

Comment:
Perhaps this `.btn-excluded` definition (which is still in use elsewhere) should be renamed?

Reply:
Yeah, I considered that, but decided to leave it.  I'm happy to change though.  Maybe just ``strikethrough`` since that's what it does.